### ISSUE TITLE
img2img often breaks

### ISSUE THREAD

**Issue by legendan** [TheLastBen/fast-stable-diffusion#1765]
Good afternoon!
 Img2img often stops working for me. Creates multiple images and the button stops being clicked, BUT txt2img still works fine.
And it's been like this for a few days. Today an error popped up (attached it), maybe this is related

![error](https://user-images.githubusercontent.com/127860844/225018543-156b8c57-501c-473d-a4fd-29778fbb96c7.jpg)

**Comment by TheLastBen** [comment]
I don't think that message causes img2umg to break, you can still generate even when it pops

**Comment by chernobog24** [comment]
I had the same issue seems to be related to a recent change in dependencies 8 days ago on hugging face. downloading the old dependencies seems to fix the issue.

**Comment by TheLastBen** [comment]
I'm trying to figure out the cause

**Comment by Dreason8** [comment]
Same error here, but with inpainting.
